sources/packages/backend/apps/queue-consumers/src/processors/schedulers/institution-integration/ece-response/_tests_/ece-response-integration.scheduler.e2e-spec.ts

```diff
@@ -71,9 +88,16 @@ describe(
         { hasIntegration: true },
         { hasIntegration: false },
       );
+
+      // Update the date sent to current date to verify newly created notifications
+      // with date sent as null.
+      await db.notification.update(
+        { dateSent: IsNull() },
+        { dateSent: getISODateOnlyString(new Date()) },
```

Comment:
`dateSent` is a regular date and date. We do not need the `getISODateOnlyString`.